test/libgit2.jl

```diff
@@ -252,6 +253,14 @@ mktempdir() do dir
                 @test typeof(e) == LibGit2.GitError
                 @test startswith(sprint(show,e),"GitError(Code:ENOTFOUND, Class:OS, Failed to resolve path")
             end
+            path = joinpath(dir, "Example.BareTwo")
+            repo = LibGit2.init(path, true)
+            try
+                #just to see if this works
```

Comment:
What do you mean with this comment? Like you're only calling `cleanup` to ensure it works?

Also, to be safe you should probably add a `catch` with something like `error("unexpected")` if you don't expect an error, or maybe a `@test_throws` if you do expect it to throw.